assets/styles/index.scss

```diff
@@ -1,5 +1,5 @@
 $icon-font-path: '~bootstrap-sass/assets/fonts/bootstrap/';
-@import '~bootstrap-sass/assets/stylesheets/bootstrap';
+@import '~bootstrap-sass/assets/stylesheets/bootstrap/';
```

Comment:
If this change doesn't break anything, you should make this change on a PR for `js-template` instead of this talk.